php.ini-development

```diff
@@ -1117,71 +1123,70 @@ odbc.allow_persistent = On
 ; https://php.net/odbc.check-persistent
 odbc.check_persistent = On
 
-; Maximum number of persistent links.  -1 means no limit.
+; Maximum number of persistent links. -1 means no limit.
 ; https://php.net/odbc.max-persistent
 odbc.max_persistent = -1
 
-; Maximum number of links (persistent + non-persistent).  -1 means no limit.
+; Maximum number of links (persistent + non-persistent). -1 means no limit.
```

Comment:
Double space after a FULL STOP is a convention for English, it is not mandatory but I'm not sure I see the problem with it.

Reply:
In the php.ini of the current master there are 169 situations where one sentence is followed by another one in the same line. In 44 cases they are separated by double space, in other 125 cases single space is used. Which is why I went for single space.

Reply:
I don't know if this is worth standardizing. A lot of people (especially older generations) prefer double spaces. You're just going to start a debate on which way is more correct.